force-app/main/freelanceCommunity/lwc/hot_myServiceAppointments/hot_myServiceAppointments.js

```diff
@@ -139,13 +139,16 @@ export default class Hot_myServiceAppointments extends LightningElement {
         }
     }
 
+    recordId;
```

Comment:
Vi trenger nok ikke å tracke variabler lenger, men fint om du legger det på, ettersom vi gjør det på alle andre variabler vi "ønsker å tracke". Vi kan jo vurder om vi vil fjerne track for alle variabler når vi går igjennom komponentene.